### PR TITLE
Fixed Some Typos

### PR DESCRIPTION
**Pull request by MahdiAljazairi**
#### Summary
Now using the correct form of "do" and "does"

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/guide/regular_expressions/assertions/index.md [hunk 1]
```diff
@@ -226,7 +226,7 @@ In the second example `^` is used both for matching at the beginning of input 
 ```js
 let fruits = ["Apple", "Watermelon", "Orange", "Avocado", "Strawberry"];
 
-// Selecting fruits that dose not start by 'A' by a /^[^A]/ regex.
+// Selecting fruits that does not start by 'A' by a /^[^A]/ regex.
```
Comment (teoli2003) [inline files/en-us/web/javascript/guide/regular_expressions/assertions/index.md:229]: As _fruits_ is plural (and therefore countable), I think it is _do_ in this case.
```suggestion
// Selecting fruits that do not start by 'A' by a /^[^A]/ regex.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Just one change and we are good to land this!

**Comment by MahdiAljazairi** [comment]
Didn't notice these ones.

File: files/en-us/web/javascript/guide/regular_expressions/assertions/index.md (hunk 1 above)
Comment (MahdiAljazairi) [reply files/en-us/web/javascript/guide/regular_expressions/assertions/index.md]: Just exploring Github 😐
Embarassing...

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/web/javascript/guide/regular_expressions/assertions/index.md (hunk 1 above)
Comment (MahdiAljazairi) [reply files/en-us/web/javascript/guide/regular_expressions/assertions/index.md]: didn't know i could comment here

**Comment by teoli2003** [comment]
Thanks a lot!

**Pull request merged**